Source/diablo.cpp

```diff
@@ -392,6 +396,10 @@ BOOL StartGame(BOOL bNewGame, BOOL bSinglePlayer)
 		run_game_loop(uMsg);
 		NetClose();
 		pfile_create_player_description(NULL, 0);
+
+		// If the player left the game into the main menu,
+		// initialize main menu resources.
+		if (gbRunGameResult) UiInitialize();

```

Comment:
Can we break the line?

Reply:
I prefer short statement to be single-line.
Our auto-formatter breaks longer ones.

Reply:
Doesn't it also rewrite this into two lines?

Reply:
Indeed it does, looks like I didn't run the formatter

I thought we had `AllowShortIfStatementsOnASingleLine` enabled :)